components/dispatcher/cmd/main.go

```diff
@@ -25,6 +26,7 @@ const (
 	DefaultKafkaConsumerOffset                     = "latest"
 	DefaultKafkaConsumerPollTimeoutMillis          = 500 // Timeout Millis When Polling For Events
 	MinimumKafkaConsumerOffsetCommitDurationMillis = 250 // Azure EventHubs Restrict To 250ms Between Offset Commits
+	HealthConfigPort                               = "8082"  // Listen Port For Liveness And Readiness Endpoints
```

Comment:
this one has to match the controller value right?  so... see if there can be a constant for it in the "right" place ; )

Reply:
common/pkg/kafka/health/constants.go is now the only place with "8082", "/healthz" and "/healthy"
controller/constants/constants.go has the channel and dispatcher delay and period as separate values now.

Reply:
I've moved the health port to a variable named HEALTH_PORT that is set for the channel and dispatcher deployments.  This is, for now, a constant in components/controller/constants/constants.go but the channel and dispatcher listen on whatever their HEALTH_PORT variable tells them to.